packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -885,8 +914,39 @@ static String interpretCircleOptions(Object o, CircleOptionsSink sink) {
     }
   }
 
+  /** Set the options in the given heatmap object to the given sink. */
+  static String interpretHeatmapOptions(Object o, HeatmapOptionsSink sink) {
```

Comment:
Can you add a javadoc describing what object o is expected to be here and below.